tools/storybook/import-wp-css-storybook.sh

```diff
@@ -5,15 +5,17 @@ STORY_BOOK_CSS_PATH="$DIR/../storybook/wordpress/css";
 TMP_DIR="$DIR/../storybook/wordpress/tmp";
 ARCHIVE_CSS_PATH="wordpress/wp-admin/css";
 ARCHIVE_IMG_PATH="wordpress/wp-admin/images";
+ARCHIVE_EDIT_SITE_PATH="wordpress/wp-includes/css/dist/edit-site/style.css";
 
 mkdir -p "$STORY_BOOK_CSS_PATH";
 mkdir -p "$TMP_DIR";
 
 function download_and_extract_css {
     curl -o "$STORYBOOK_WORDPRESS_DIR/wordpress-latest.zip" https://wordpress.org/nightly-builds/wordpress-latest.zip;
-    unzip -qq "$STORYBOOK_WORDPRESS_DIR/wordpress-latest.zip" "$ARCHIVE_CSS_PATH/*" "$ARCHIVE_IMG_PATH/*" -d "$TMP_DIR";
+    unzip -qq "$STORYBOOK_WORDPRESS_DIR/wordpress-latest.zip" "$ARCHIVE_CSS_PATH/*" "$ARCHIVE_IMG_PATH/*" "$ARCHIVE_EDIT_SITE_PATH" -d "$TMP_DIR";
     rsync -a "$TMP_DIR/$ARCHIVE_CSS_PATH" "$STORYBOOK_WORDPRESS_DIR";
     rsync -a "$TMP_DIR/$ARCHIVE_IMG_PATH" "$STORYBOOK_WORDPRESS_DIR";
+    rsync -a "$TMP_DIR/$ARCHIVE_EDIT_SITE_PATH" "$STORYBOOK_WORDPRESS_DIR/css/edit-site.css";
     rm -r "$TMP_DIR";
```

Comment:
Small item, although was already present prior, is that we should really delete the `wordpress-latest.zip` as well.
Oddly enough I never noticed the `zip` file before until testing now.

Reply:
Fixed!